.github/workflows/cibuildwheel.yml

```diff
@@ -1,3 +1,15 @@
+# This CI run builds wheels using the cibuildwheel infrastructure.
+# This means it:
+# - Uses as old dependencies/infrastructure as possible:
+#   - An old manylinux docker container on Linux
+#   - An old MACOSX_DEPLOYMENT_TARGET on macOS
+#   - An old SYSTEM_VERSION target on Windows
+# - Builds OpenMEEG
+# - Creates wheels
+# - Delocates them
+# - Uploads them to TestPyPI (main) or PyPi (releases)
```

Comment:
> Are these specific to python ?

This comment is meant to answer this part. More or less, yes.

*But*, I think we could use it to take care of https://github.com/openmeeg/openmeeg/issues/495 as well. I think if we `-DENABLE_APPS=ON` in the wheel build instead of `-DENABLE_APPS=OFF`, we'll have all the files we need and it's "just" a packaging problem at that point.